src/collection-facets.ts

```diff
@@ -9,11 +9,18 @@ import {
 } from 'lit';
 import { customElement, property, state } from 'lit/decorators.js';
 import { repeat } from 'lit/directives/repeat.js';
-import { Aggregation, Bucket } from '@internetarchive/search-service';
-import '@internetarchive/histogram-date-range';
+import {
+  Aggregation,
+  Bucket,
+  SearchServiceInterface,
+} from '@internetarchive/search-service';
+// import '@internetarchive/histogram-date-range';
```

Comment:
Q: what happened here? not working properly?

Reply:
Hmm :( the ia-activity-indicator compenent making noise here as again. its saying already defined in console error.

Reply:
yah I totally am hitting that as well.
it is bc histogram is holding an incompatible version of the activity indicator.
I am upgrading it in my PR that I will merge today. once u rebase master with my fix, it should be cleared.